tests/pipelines/blipdiffusion/test_blipdiffusion.py

```diff
@@ -0,0 +1,178 @@
+import gc
```

Comment:
Missing licensing. Also the name of the file should start with "test_pipeline".

Reply:
@sayakpaul I can't see the other test files following that convention, https://github.com/huggingface/diffusers/tree/main/tests/pipelines/kandinsky

Reply:
Ah okay to ignore then. But the licensing is missing.